web/styles/informational_overlays.css

```diff
@@ -103,6 +108,10 @@
     margin-bottom: 10px !important;
 }
 
+& p > time > span.timestamp-content-wrapper {
```

Comment:
There must be a simpler selector than this to get the needed change here. `.timestamp-content-wrapper` is plenty specific; it just needs to be isolated to the message-formatting table somehow.

Reply:
Thank you for your review. I’ve updated the CSS selector as follows:
```
#message-formatting p:has(time) {
    margin-top: var(--markdown-interelement-space-px);
}
```
This change should address the feedback and achieve the desired result.
I look forward to your review of the updated changes.

Reply:
That's an improvement, for sure. I still think it would be good to open a #design discussion about removing inter-element spacing on all of the modal examples, and then this brittle selector would become unnecessary entirely.

Reply:
Thank you for your review. In response, I've opened a [discussion](https://chat.zulip.org/#narrow/stream/101-design/topic/layout.20bugs.20in.20the.20help.20modal) in the design channel and would like to hear your feedback on this topic.